src/components/views/messages/MJitsiWidgetEvent.tsx

```diff
@@ -17,31 +17,41 @@ limitations under the License.
 import React from 'react';
 import { MatrixEvent } from "matrix-js-sdk/src/models/event";
 import { _t } from "../../../languageHandler";
+import WidgetStore from "../../../stores/WidgetStore";
+import { WidgetType } from "../../../widgets/WidgetType";
 
 interface IProps {
     mxEvent: MatrixEvent;
 }
 
-interface IState {
-}
-
-export default class MJitsiWidgetEvent extends React.PureComponent<IProps, IState> {
+export default class MJitsiWidgetEvent extends React.PureComponent<IProps> {
     constructor(props) {
         super(props);
-        this.state = {};
     }
 
     render() {
         const url = this.props.mxEvent.getContent()['url'];
         const prevUrl = this.props.mxEvent.getPrevContent()['url'];
         const senderName = this.props.mxEvent.sender?.name || this.props.mxEvent.getSender();
 
+        // XXX: We are assuming that there will only be one Jitsi widget per room, which isn't entirely
+        // safe but if there's more than 1 the user will be super confused anyways - the copy doesn't
+        // need to concern itself with this.
+        const roomInfo = WidgetStore.instance.getRoom(this.props.mxEvent.getRoomId());
```

Comment:
shouldn't you know the widgetId from the state_key of this event? So you could lookup isPinned(state_key)

Reply:
I suppose, but that's not as fun as leaving lazy comments :p